### PR TITLE
Finding filters

### PR DESCRIPTION
**Pull request by cloudsmart**
This pull request is to merge a few minor changes I've made in my local branch with the master branch.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,6 +71,8 @@ builds/
 #
 # example.tfvars
 **/terraform.tfvars
+**/backend.tf
+**/.terraform.lock.hcl
```
Comment (sheaphillips) [inline .gitignore]: terraform lock file file should be included in git and therefor *not* referenced in .gitignore.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,66 @@
...
+You would need to set up Webhook URL's for the above RC channels and add the URL's to `IncomingWebHookUrl` in a *tfvars* file prior to deployment. The *tfvars* file should contain the following variables:
+
+- aws_region = The AWS region you want to deploy the app to, e.g. `ca-central-1`
+- IncomingWebHookUrl = `"BUDGET=<Webhook URL for aws-budget>,LOW=<Webhook URL for aws-security-low>,MEDIUM=<Webhook URL for aws-security-medium>,HIGH=<Webhook URL for aws-security-high>,CRITICAL=<Webhook URL for aws-security-critical>"`
+- LambdaEnvLogLevel = Lambda event log level, e.g. `INFO`
+- LambdaTimeout = Lambda timeout in second, e.g. `10`
+
+## Build, Packaging and Testing Locally
+
+- **Build**: run ```sam build --use-container -b ../../builds``` within the lambdas/rocketchat-notification folder
```
Comment (sheaphillips) [inline README.md:46]: As an enhancement, we should attempt to make it simpler to run/deploy by incorporating calls to `sam` and zip operation into Terraform.

**Review by sheaphillips — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (cloudsmart) [reply .gitignore]: Done! change is included in the last commit.

File: README.md (hunk 2 above)
Comment (cloudsmart) [reply README.md:46]: Working on it...

Comment (sheaphillips) [reply README.md:46]: are you intending to complete this as part of the current ticket (#629)?

Comment (sheaphillips) [reply README.md:46]: I'd say if you have time, pls proceed with the enhancement.

**Review by sheaphillips — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cloudsmart) [reply README.md:46]: Yes, I'll try my best!

**Comment by wrnu** [comment]
No longer relevant.

**Pull request closed without merging**